plugin/index.ts

```diff
@@ -5,6 +5,7 @@ import traverse from '@babel/traverse';
 import { transformSync } from '@babel/core';
 import * as fs from 'fs';
 import * as convertSourceMap from 'convert-source-map';
+import reanimatedPluginVersion from './package.json';
```

Comment:
The original idea was to inject the version from `react-native-reanimated/package.json`. This way we don't need the `version` field inside `plugin/package.json` and we don't need to update it accordingly with each version bump. 

I'm okay with having a separate version in `plugin/package.json` as long as we have some automatic check on CI that ensures that the values are equal.

Reply:
That is a good idea.

About the CI check - we already have it, it runs `checkVersion.ts`.

Reply:
```suggestion
import reanimatedPluginVersion from '../package.json';
```